client/actions/characterActions.js

```diff
@@ -1,12 +1,11 @@
+import characterSchema from './schemas/characterSchema'
+
 const addCharacter = () => (dispatch) => {
   return dispatch({
     type: 'CREATE_CHARACTER',
-    data: {
-      name: 'Mikely Gregor',
-      race: 'Human',
-      class: 'Monk',
-    }
-  })
+    schema: characterSchema,
+    query: '{ name, race, class }'
```

Comment:
is the schema necessary for the FE. This is definitely needed on the BE as there is where the graph will be greated, FE from my point of just needs to POST the query. I could however be wrong by the way.

Reply:
sorry I have naming off here, this is get character

Reply:
however, from my understanding that query line is what i specify to say the information i want